lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -433,7 +433,7 @@ class TestAuthModule:
     encode_auth_token() since they are slated for removal and are covered well
     by other parts of the unit testing for the server code.
 
-    It does, however, verify the verify_auth_internal() method because we need
+    It does, however, verify the verify_auth() method because we need
     it to function properly until the use of an internal user is removed.
```

Comment:
🤔  At this point, haven't we removed the use of an "internal user"?  And, isn't it clear by now that we're not intending to remove `get_current_user_id()`?  And, doesn't this PR remove `encode_auth_token()`?.

I think we need to touch up lines 432-437.